packages/app-store-cli/src/core.ts

```diff
@@ -112,6 +112,7 @@ export const BaseAppFork = {
       isTemplate,
       // Store the template used to create an app
       __template: template,
+      dirName: slug,
```

Comment:
No need to set it in config.json. I want to deprecate that.
seed-app-store.config.json already has it and seeding uses seed-app-store.config.json only. So, it gets the dirName https://github.com/calcom/cal.com/blob/5a94228df61e34d0948f067e7b5ac1fd6060c7ab/packages/prisma/seed-app-store.ts#L308 https://github.com/calcom/cal.com/blob/5a94228df61e34d0948f067e7b5ac1fd6060c7ab/packages/app-store-cli/src/core.ts#L164

Reply:
I noticed that `dirName` wasn't being added when enabling apps from the admin apps panel. Do we need the `dirName` column if it's just the app slug?

Reply:
Certain Legacy apps have different dirname than slug, so yes for now.